internal/pubsub/acks_test.go

```diff
@@ -176,7 +176,7 @@ func TestTooManyAcksForASingleBatchGoIntoMultipleBatches(t *testing.T) {
 
 func TestAckDoesNotBlock(t *testing.T) {
 	ctx, cancel := context.WithCancel(context.Background())
-	m := &driver.Message{}
+	m := &driver.Message{AckID: 0}
```

Comment:
What does this have to do with removing ackIDBox?

Reply:
The reflect package chokes on nil interface values. I don't fully understand why, but AckIDs will never be nil in practice so it shouldn't matter except for tests.

Reply:
Probably worth a comment.

Reply:
Done.